### ISSUE TITLE
Add `-Ordered` parameter info to documentation

### ISSUE THREAD

**Issue by frankvaneykelen-work** [cloudbase/powershell-yaml#115]
I had an issue with an azure DevOps build YAML file that was generated via loading it from a template (ConvertFrom-Yaml), processing it, and saving it (ConvertTo-Yaml). In the resulting file the properties of nodes in the YAML were in a different order than in the original template, causing the build configuration to be rejected ("The first property must be 'repository'").

The fix was easy - running it with the `-Ordered` parameter
```
ConvertFrom-Yaml -Ordered
```

I discovered it via running `get-help ConvertFrom-Yaml` but it would be nice if this parameter was covered in the docs too. Also because the other two (`-UseMergingParser` and `-AllDocuments`) are already mentioned.

**Comment by gabriel-samfira** [comment]
You're right. Documentation is sorely missing. I'll allocate some time to write proper doc strings for each function and options. and also update the README.
